test/ctia/http/generative/atom_store_spec.clj

```diff
@@ -0,0 +1,38 @@
+(ns ctia.http.generative.atom-store-spec
```

Comment:
This is kind of new and weird.  The tests are grouped by the test setup rather then by the entity; They are grouped by the store and not by the entity.  This makes a lot of sense for generative tests since each defspec below might trigger hundreds of API calls.  It is best to have a :once fixture at the top and not do any :each for each property being tested.  As a consolation, this is all just glue code and the specs are defined in one place.